qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -987,9 +1014,14 @@ trash_move() {
 trash_restore() {
     local cluster=$1
     local pool=$2
-    local image_id=$3
+    local image=$3
+    local mode=${4:-${RBD_MIRROR_MODE}}
 
+    local image_id=$(get_image_id ${cluster} ${pool} ${image})
     rbd --cluster=${cluster} trash restore ${pool}/${image_id}
+    if [ "${mode}" = "snapshot" ]; then
```

Comment:
It is still confusing (why it does it apply for "snapshot" mode and not for journal?). I suggested to check for MIRROR_POOL_MODE. I think the check should be the same as in `create_image_and_enable_mirror`.

Reply:
I will fix it as suggested.